### DISCUSSION TITLE
Plugin bypass in DAW

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1083] [Help]
When a plugin is disabled in Studio One it is not loaded of course in AG.

But when I bypass a plugin in Studio one (hard bypass), depending on the session, sometimes the plugin is loaded by AG on the server, sometimes not on other sessions. 
Is there an option to handle that in AG ?

Config : Studio one 6 M1 native.

**Comment by apohl79** [comment]
Maybe S1 is not always loading a bypassed plugin. In this case AG would also not load a plugin on the server. But I have no experience with S1. So no idea really.
